### PR TITLE
Update Java EE Quickstart article for RS256

### PR DESCRIPTION
**Pull request by jimmyjames**
Accompanies sample application PR found here: https://github.com/auth0-samples/auth0-java-ee-sample/pull/1

### CODE REVIEW

File: articles/quickstart/webapp/java-ee/_includes/_login.md [hunk 1]
```diff
@@ -119,7 +119,16 @@ public class Auth0JwtIdentityStore implements IdentityStore {
 
 If the `credential` is an `Auth0Credential`, the calling user is authenticated and valid, so a `CredentialValidationResult` created with the credential is returned to indicate success. If it is not an `Auth0Credential`, return `CredentialValidationResult.NOT_VALIDATED_RESULT`.
 
-Before implementing the `HttpAuthenticationMechanism` interface that will use all these collaborators, create a bean that will provide a configured instance of the `AuthenticationController` from the Auth0 Java MVC SDK, used to build the authorize URLs and handle the token exchange to authenticate users:
+Before implementing the `HttpAuthenticationMechanism` interface that will use all these collaborators, create a bean that will provide a configured instance of the `AuthenticationController` from the Auth0 Java MVC SDK. The `AuthenticationController` is used to build the authorize URL where users will login, and handle the token exchange to authenticate users.
```
Comment (damieng) [inline articles/quickstart/webapp/java-ee/_includes/_login.md]: I think you mean "authentication URL" here?

File: articles/quickstart/webapp/java-ee/_includes/_login.md [hunk 2]
```diff
@@ -119,7 +119,16 @@ public class Auth0JwtIdentityStore implements IdentityStore {
...
+Before implementing the `HttpAuthenticationMechanism` interface that will use all these collaborators, create a bean that will provide a configured instance of the `AuthenticationController` from the Auth0 Java MVC SDK. The `AuthenticationController` is used to build the authorize URL where users will login, and handle the token exchange to authenticate users. 
+
+* If your Auth0 Application is configured to use the **RS256 signing algorithm** (the default when creating a new Auth0 Application), you need to configure a `JwkProvider` to fetch the public key used to verify the token's signature. See the [jwks-rsa-java repository](https://github.com/auth0/jwks-rsa-java) to learn about additional configuration options.
+* If your Auth0 Application is configured to use the **HS256 signing algorithm**, there is no need to configure the `JwkProvider`.
+
+::: note
+To learn more about the available signing algorithms, refer to the [documentation](https://auth0.com/docs/tokens/concepts/signing-algorithms).
+:::
+
+The sample below hows to configure the `AuthenticationController` for use with the **RS256 signing algorithm**:
```
Comment (damieng) [inline articles/quickstart/webapp/java-ee/_includes/_login.md]: Should "hows" be "shows"?

File: articles/quickstart/webapp/java-ee/_includes/_setup.md [hunk 3]
```diff
@@ -76,7 +76,7 @@ Your Java EE application needs some information in order to authenticate users w
 </env-entry>
 ```
 
-This information will be used to configure the Auth0 Java MVC SDK to allow authentication with users. To learn more about the SDK, including its various configuration options, see the [README](https://github.com/auth0/auth0-java-mvc-common/blob/master/README.md) of the SDK.
+This information will be used to configure the **auth0-java-mvc-commons** library to enable users to login to your application. To learn more about the library, including its various configuration options, see the [README](https://github.com/auth0/auth0-java-mvc-common/blob/master/README.md) of the library.
```
Comment (damieng) [inline articles/quickstart/webapp/java-ee/_includes/_setup.md:79]: Maybe "this libraries README" or "see the README for that library".  "see the README of the library" sounds awkward.

**Review by damieng — CHANGES_REQUESTED** [review]

File: articles/quickstart/webapp/java-ee/_includes/_login.md (hunk 1 above)
Comment (jimmyjames) [reply articles/quickstart/webapp/java-ee/_includes/_login.md]: Actually it should be "authorization url" (https://auth0.com/docs/flows/guides/auth-code/add-login-auth-code#example-authorization-url). I'll update it.

File: articles/quickstart/webapp/java-ee/_includes/_setup.md (hunk 3 above)
Comment (jimmyjames) [reply articles/quickstart/webapp/java-ee/_includes/_setup.md:79]: How about just removing the word "README" entirely? How about "the library's documentation"?

**Review by damieng — APPROVED** [review]

**Pull request merged**